algebra/tests/quat/various.c

```diff
@@ -380,3 +384,138 @@ TEST_GROUP_RUNNER(group_quat_dif)
 	RUN_TEST_CASE(group_quat_dif, quat_dif_std);
 	RUN_TEST_CASE(group_quat_dif, quat_dif_biggerValues);
 }
+
+
+/* ##############################################################################
+ * ----------------------        quat_mlt tests       -----------------------
+ * ############################################################################## */
+
+
+TEST_GROUP(group_quat_mlt);
+
+
+TEST_SETUP(group_quat_mlt)
+{
+}
+
+
+TEST_TEAR_DOWN(group_quat_mlt)
+{
+}
+
+
+TEST(group_quat_mlt, quat_mlt_quatMltTable)
+{
+	quat_t qA, qI, qJ, qK, nqA, nqI, nqJ, nqK, qZero = { 0 };
+
+	qA = qI = qJ = qK = nqA = nqI = nqJ = nqK = qZero; /* zero all quaternions */
+	qA.a = qI.i = qJ.j = qK.k = 1.0f;                  /* set all positive unitary quaternions */
+	nqA.a = nqI.i = nqJ.j = nqK.k = -1.0f;             /* set all negative unitary quaternions */
+
+
+	quat_t Res;
```

Comment:
Double newline + declaration goes first.
```suggestion
	quat_t qA, qI, qJ, qK, nqA, nqI, nqJ, nqK;
	quat_t Res, qZero = { 0 };

	qA = qI = qJ = qK = nqA = nqI = nqJ = nqK = qZero; /* zero all quaternions */
	qA.a = qI.i = qJ.j = qK.k = 1.0f;                  /* set all positive unitary quaternions */
	nqA.a = nqI.i = nqJ.j = nqK.k = -1.0f;             /* set all negative unitary quaternions */
```